src/content/docs/datahub/datahub-invite-collaborators.mdx

```diff
@@ -1,46 +1,49 @@
 ---
-title: Invite collaborators to your ARC
+title: How to invite collaborators to your ARC
```

Comment:
Maybe "Working Together"

Reply:
Working together could mean "user groups" or "invite people". 

The datahub guides were mostly split into small single tasks. But yes, we can collect them all together again.